lib/board.rb

```diff
@@ -33,18 +30,31 @@ def place(ship, coordinates)
   end
 
   def render(debug = false)
+    columns = [' ', @columns, "\n"].flatten.join(' ')
```

Comment:
[ to ] creates an array with 3 elements:

1 - ' ' used for alignment/padding 
2 - @columns
3 - "\n" newline character

flatten turns an array into a single level array. 
[' ', 1, 2, 3, 4, "\n"]

join(' ') concatenates all elements not a single string separating them with a space. 
"   1 2 3 4 \n"